docs/src/components/app/style.scss

```diff
@@ -68,14 +68,46 @@
     padding: 2px;
     margin: 2px;
   }
+  .code-snippet {
+    font-variant: common-ligatures;
```

Comment:
Font display : swap?

Reply:
It has to go into the `@font-face`, but I've inserted it.